src/components/layout/header/header.tsx

```diff
@@ -0,0 +1,30 @@
+import * as F from "./header-styles"
+import logo from "../../../imgs/white-logo.png"
+import { useNavigate } from "react-router-dom"
```

Comment:
외부모듈을 가장 위로, 외부 모듈과 내부 모듈에는 한칸 띄어주세요
```suggestion
import { useNavigate } from "react-router-dom"

import * as F from "./header-styles"
import logo from "../../../imgs/white-logo.png"
```

Reply:
네 ㅎㅎ